### PR TITLE
docs(readme): add npx use examples

### LINKED ISSUE #502: Update docs to use npx to install locally
Hi! With the latest node and NPM version I believe that we can do:
`npx commitizen init cz-conventional-changelog --save-dev --save-exact`

I'll be happy to create a PR if this is something we want to update the docs.

### PR DESCRIPTION
**Pull request by giocodes**
closes #502

### CODE REVIEW

**Comment by giocodes** [comment]
Good point @jimthedev, thanks for that. 

I've pushed an update.

File: README.md [hunk 1]
```diff
@@ -101,7 +113,12 @@ Installing and running Commitizen locally allows you to make sure that developer
 
 Install Commitizen with `npm install --save-dev commitizen`.
 
-Once you have Commitizen installed as a local dev dependency you can execute `./node_modules/.bin/commitizen` or `./node_modules/.bin/git-cz` in order to actually use the commands.
+On **NPM 5.2+** you can [use `npx`](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b) to initialize the conventional changelog adapter:
+```
+npx commitizen init cz-conventional-changelog --save-dev --save-exact
+```
+
+For **previous versions of Node (<= 9) ** you can execute `./node_modules/.bin/commitizen` or `./node_modules/.bin/git-cz` in order to actually use the commands.
```
Comment (jimthedev) [inline README.md]: Did you mean before npm 5.2?

Comment (giocodes) [reply README.md]: Yes, thank you!

**Comment by jimthedev** [comment]
Thank you!

**Pull request merged**